### PR TITLE
Feature: help command and embed factory functionality

### PR DESCRIPTION
**Pull request by Kevnkkm**
Pls check for noob errors because I don't code Discord bots with python thanks

### CODE REVIEW

**Review by probablyjassin — COMMENTED** [review]
seems good

File: cogs/help.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                         "https://raw.githubusercontent.com/mk8dx-yuzu/mk8dx-yuzu.github.io/main/public/favicon/ms-icon-310x310.png",
+                         commands,
+                         {"text": "Yuzu Online", "icon_url": "https://images-ext-1.discordapp.net/external/ymL8nMKRGEJwQZNCLRuCAbeHxt3n3HYA0XTD-JUW4m4/https/cdn.discordapp.com/icons/1084911987626094654/a_f51d88cf4421676675437f9cf4fbbff6.gif"},
+                         color=Color.blue(),
+                         inline=False)
+            
+                        # TODO: Make the footer image and text take the guild's icon and name instead of hard coded values ↑
+            await ctx.respond(embed=embed)
+
+        @commands.command(name="lan")
```
Comment (probablyjassin) [inline cogs/help.py]: tbh i dont wanna include these commands that send gifs, keep em on the current bot

Comment (probablyjassin) [reply cogs/help.py]: oh did u just copy paste them and forgor to remove them anyway, since u didnt add the files anyway

Comment (Kevnkkm) [reply cogs/help.py]: Nah I added them after I was done, I thought we would move them to the new bot but it makes sense to keep it on the "utility" bot. And I wasn't sure where the files were so I just kept the code

**Pull request merged**